dev_build.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+###############################################################################
+# Copyright 2018 David Hopper(davidhopper2003@hotmail.com). All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###############################################################################
+
+current_docker=$(docker ps | grep 'apolloauto/apollo')
```

Comment:
I think this file duplicates the functions in apollo_docker.sh

Reply:
I carried it out myself. I didn't notice the existence of the "apollo_docker.sh" file. Because Apollo's help documentation doesn't mention it. With this file, I think my script file is unnecessary. I will change my code an submit again. Thank you for your valuable suggestions.